src/Tutor.Web/Startup.cs

```diff
@@ -87,14 +94,58 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<WorkspaceOptions>(Configuration.GetSection(WorkspaceOptions.ConfigKey));
             services.AddScoped<IWorkspaceCreator, NoWorkspaceCreator>();
             services.AddScoped<ILearnerRepository, LearnerDatabaseRepository>();
+            services.AddScoped<IJwtService, JwtService>();
+            services.AddScoped<IRefreshTokenValidator, RefreshTokenValidator>();
 
             services.AddScoped<IAuthProvider, KeycloakAuthProvider>();
+            services.AddScoped<IAuthService, AuthService>();
+
+            services.AddAuthorization(options =>
+            {
+                options.AddPolicy("learnerPolicy", policy =>
+                    policy.RequireRole("learner"));
+            });
 
             if (!bool.Parse(Environment.GetEnvironmentVariable("KEYCLOAK_ON") ?? "false")) return;
             AuthenticationConfig(services);
             AuthorizationConfig(services);
         }
 
+        private static void SetupJwtService(IServiceCollection services)
```

Comment:
We have AuthenticationConfig, AuthorizationConfig, SetupJwtService, and services.AddAuthorization inlined in the main method.

Can we clean this up? Move everything related to A&A into a separate, centralized method?